conjur-api/Client.Methods.cs

```diff
@@ -5,13 +5,17 @@
 //     Conjur Client methods delegating to entity-specific classes.
 // </summary>
 
+using System.Collections.Generic;
+
 namespace Conjur
 {
     /// <summary>
     /// Entity-specific methods for the Client facade.
     /// </summary>
     public partial class Client
     {
+        private uint limitSearchVarListsReturned = 1000;
```

Comment:
Let's make this a CONST (read-only, ALL_CAPS)

Reply:
Also - is 1000 the best practice? @dividedmind

Reply:
Should be fine. We might consider adding a hint parameter as to the chunk size, but YAGNI - I'd postpone this until and unless we find it necessary.